client/src/Components/Review.jsx

```diff
@@ -36,15 +36,15 @@ class Review extends React.Component {
               </div>
               <div className="reviews">
                 <div id="inner">
-                  <div id="review_title">{review.review.title}</div>
-                  <div id="review">{review.review.review}</div>
+                  <div id="review_title">{review.title}</div>
+                  <div id="review">{review.review}</div>
```

Comment:
HA this is better than `review.review.review`, but you still may want to consider a new name for the parent object.

Reply:
Changed to singleReview for clarity!